lib/constants/skip-reason.ts

```diff
@@ -0,0 +1,43 @@
+export default {
+  UNKNOWN: 'unknown',
+  NO_VERSION: 'no-version',
+  NO_SOURCE_MATCH: 'no-source-match',
+  UNSUPPORTED_REMOTE: 'unsupported-remote',
+  GIT_PLUGIN: 'git-plugin',
+  INVALID_VERSION: 'invalid-version',
+  PATH_DEPENDENCY: 'path-dependency',
+  GIT_DEPENDENCY: 'git-dependency',
+  INVALID_DEPENDENCY_SPECIFICATION: 'invalid-dependency-specification',
+  UNSUPPORTED: 'unsupported',
+  ANY_VERSION: 'any-version',
+  UNKNOWN_VERSION: 'unknown-version',
+  UNSUPPORTED_VERSION: 'unsupported-version',
+  PLACEHOLDER_URL: 'placeholder-url',
+  LOCAL_DEPENDENCY: 'local-dependency',
+  INVALID_URL: 'invalid-url',
+  NO_REPOSITORY: 'no-repository',
+  LOCAL_CHART: 'local-chart',
+  UNSUPPORTED_CHART_TYPE: 'unsupported-chart-type',
+  UNKNOWN_REGISTRY: 'unknown-registry',
+  UNSUPPORTED_URL: 'unsupported-url',
+  INVALID_SHA256: 'invalid-sha256',
+  NAME_PLACEHOLDER: 'name-placeholder',
+  VERSION_PLACEHOLDER: 'version-placeholder',
+  NON_HEX_DEPTYPES: 'non-hex depTypes',
+  INVALID_NAME: 'invalid-name',
+  INVALID_VALUE: 'invalid-value',
+  UNKNOWN_ENGINES: 'unknown-engines',
+  UNKNOWN_VOLTA: 'unknown-volta',
+  FILE: 'file',
+  EMPTY: 'empty',
+  UNVERSIONED_REFERENCE: 'unversioned-reference',
+  NOT_A_VERSION: 'not-a-version',
+  IGNORED: 'ignored',
+  FILE_DEPENDENCY: 'file-dependency',
+  MULTIPLE_CONSTRAINT_DEP: 'multiple-constraint-dep',
+  LOCAL: 'local',
+  NO_SOURCE: 'no-source',
+  INTERNAL_PACKAGE: 'internal-package',
+  DISABLED: 'disabled',
+  UNSUPPORTED_VALUE: 'unsupported-value',
```

Comment:
@ViralRuparel can you sort these alphabetically? Will make it easier to unify them later. @viceice any thoughts also on whether we should prefix them all (e.g. `SKIP_REASON_LOCAL`) or keep as here?

Reply:
sure, let me do it.

Reply:
I think we don't need SKIP_REASON as a prefix to each one of them as when we import and use it, we already use it as e.g. `skipReason.UNDEFINED` which makes it clear that it is a skipReason constant.